### DISCUSSION TITLE
بما أن هذا تطبيق مفتوح المصدر

### DISCUSSION THREAD

**Discussion by Al-fata-tayeb** [TeamNewPipe/NewPipe#10635] [Questions]
هل يمكنني إستعماله في نقله إلى نظام لينكس ديبيان ؟ و كيف يمكنني إستعمال مصدركم؟

**Comment by opusforlife2** [comment]
Use google translate. English only.

**Reply by Al-fata-tayeb** [reply]
لماذا ؟؟؟؟؟
